tests/system/test_response_metadata.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+
+from google import showcase
+
+
+@pytest.mark.parametrize(
+    "transport,response_metadata",
```

Comment:
How do the interceptors used int his file know about the response metadata they should be injecting? Or  is this real response metadata from Showcase? I'm not finding it by doing a search n the Showcase repo.